packages/editor/src/store/reducer.js

```diff
@@ -136,6 +136,7 @@ export const editor = flow( [
 		ignoreTypes: [
 			'RESET_POST',
 			'UPDATE_POST',
+			'RESET_EDITOR_SELECTION',
```

Comment:
Does this action type exist?

Reply:
Looks like a leftover, I'll remove this and the withHistory changes.